pkg/network/encoding/encoding.go

```diff
@@ -64,7 +64,7 @@ func modelConnections(conns *network.Connections) *model.Connections {
 	cfgOnce.Do(func() {
 		agentCfg = &model.AgentConfiguration{
 			NpmEnabled: config.Datadog.GetBool("network_config.enabled"),
-			TsmEnabled: config.Datadog.GetBool("service_monitoring_config.enabled"),
+			UsmEnabled: config.Datadog.GetBool("service_monitoring_config.enabled"),
```

Comment:
I had to copy this from #15143 because of an incompatible change made in DataDog/agent-payload#223.